tests/qiskit_convert_test.py

```diff
@@ -865,6 +867,21 @@ def test_controlled_unitary_conversion() -> None:
     assert np.allclose(u_qc, u_tkc)
 
 
+def test_qcontrol_box_conversion_to_qiskit() -> None:
```

Comment:
I think I will add tests for a wider range of controlled operations

Reply:
Yes please.

Reply:
I've fixed the conversion of parametrised gates in https://github.com/CQCL/pytket-qiskit/pull/391/commits/b42946e54230c61bb9b65037f5ae6e31f7b19ab2 and added some controlled parametrised gates in https://github.com/CQCL/pytket-qiskit/pull/391/commits/c1b315cbacf7e66de9cf34cbf7c41358c2652a98